extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeMixPlaylistExtractor.java

```diff
@@ -30,8 +30,8 @@
 import static org.schabi.newpipe.extractor.services.youtube.YoutubeParsingHelper.toJsonArray;
 
 /**
- * A YoutubePlaylistExtractor for a mix (auto-generated playlist). It handles urls in the format of
- * "youtube.com/watch?v=videoId&list=playlistId"
+ * A {@link YoutubePlaylistExtractor} for a mix (auto-generated playlist). It handles URLs in the format of
+ * <code>youtube.com/watch?v=videoId&list=playlistId"</code>
```

Comment:
```suggestion
 * <code>youtube.com/watch?v=videoId&list=playlistId</code>
```
Looks like I did not remove that in my suggestion.

Reply:
Also use `{@code something}` instead of `<code>something</code>`